xml/decoders/Doehler_Haass_fw3.04_DH05-10C_12-16-18-21-22A.xml

```diff
@@ -117,23 +119,23 @@
           <protocol>motorola</protocol>
         </protocols>
       </model>
-	  <model model="DH18A v3.04 (firmware 3.04+)" lowVersionID="54" highVersionID="54" numOuts="6" numFns="16" productID="DH18A_3_04" comment="DH18A with update from 31 May 2014" maxInputVolts="30V" maxMotorCurrent="1.5A" maxTotalCurrent="1.5A" connector="Next18">
+	  <model model="DH18A v3.04 (firmware 3.04+)" replacementModel="DH18A (firmware 3.12.050+)" replacementFamily="query:Train Decoders (2020)" lowVersionID="54" highVersionID="54" numOuts="6" numFns="16" productID="DH18A_3_04" comment="DH18A with update from 31 May 2014" maxInputVolts="30V" maxMotorCurrent="1.5A" maxTotalCurrent="1.5A" connector="Next18">
         <output name="1" label="Front|Light" maxcurrent="150mA"/>
-		<output name="2" label="Rear|Light" maxcurrent="150mA"/>
-		<output name="3" label="AUX|1" maxcurrent="300mA"/>
-		<output name="4" label="AUX|2" maxcurrent="300mA"/>
-		<output name="5" label="AUX|3" maxcurrent="NC - Logic Level"/>
-		<output name="6" label="AUX|4" maxcurrent="NC - Logic Level"/>
+		    <output name="2" label="Rear|Light" maxcurrent="150mA"/>
+		    <output name="3" label="AUX|1" maxcurrent="300mA"/>
+		    <output name="4" label="AUX|2" maxcurrent="300mA"/>
+	      <output name="5" label="AUX|3" maxcurrent="NC - Logic Level"/>
+	      <output name="6" label="AUX|4" maxcurrent="NC - Logic Level"/>
```

Comment:
@PierreMc 
I now see that there are a number of TAB characters in these files. It would be great if you could convert these TABs to spaces. The problem with TABs is that different editors has different settings for how many spaces a TAB character represent. The TABs where there probably before your changes, but it's good if they are removed.

Reply:
Thanks, corrected in 694dbd5697b666b2a3acad4b0bb78695ae980aa4